composer.json

```diff
@@ -5,15 +5,16 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "phpspec/phpspec": "~2.5.0",
+        "php": "^5.6|^7.0",
```

Comment:
I know this is the minimum correct version etc, but realistically this will not work for a significant number of Magento 1 projects. I would prefer if this only reflected the language features used by the extension rather than dictating version requirements to other people's projects.